src/main.py

```diff
@@ -0,0 +1,114 @@
+import cv2
+import imutils
+import numpy as np
+
+# global variables
+BG = None
+
+def run_avg(image: np.ndarray, aWeight: float)->None:
```

Comment:
Leave a whitespace before and after the "arrow" `(->)` when specifying type annotations. 
Ex: 
```python
def run_avg(image: np.ndarray, a_weight: float) -> None:
```